Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tantivy"
-version = "0.13.2"
+version = "0.14.0"
```

Comment:
I think the tantivy upgrade should be a separate or is there something this PR needs from 0.14?

Reply:
I'm using this branch on prod, I can do a new PR for the 0.14 if its better.

Reply:
Yes please, separate PR for the 0.14 bump would be nice.